internal/pkg/docker/orchestrator/orchestrator.go

```diff
@@ -475,6 +475,7 @@ func (o *Orchestrator) waitForContainerToStart(ctx context.Context, id string) e
 		case err != nil:
 			return fmt.Errorf("check if %q is running: %w", id, err)
 		case isRunning:
+			log.Successf("Successfully started container %s\n", id)
```

Comment:
Not for this PR, but some time later after this is merged - 

I feel like it'd be easier to refactor the code in the future if we add a logger to Orchestrator:
```yaml
type Orchestrator struct {
  logger log.Logger
}

func New() *Orchestrator {
  return &Orchestrator { logger: log.New() }
}

func (o *Orchestrator) waitForContainerToStart(ctx context.Context, id string) error {
   o.logger.Successf("...")
}
```

This would be easier for us to a) mute the logger by creating a no-op loggeer in New() b) create a logger that logs to an open file instead of stdout/err and c) refactor the codes, because we can easily locate all the logging by "Go to reference" of the `logger` (vs. now we need to do a global search on the keyword "log")